pkg/templates/dockersock/template.go

```diff
@@ -0,0 +1,53 @@
+package dockersock
```

Comment:
This doesn't need to be its own template. You can just write a check that uses the volume-mount template you created with a value of "docker.sock"? (Once you make that support regex)

Reply:
good suggestion. removed `docker-sock` template. changed `docker-sock` check to use `host-mount` template.